projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -118,30 +109,23 @@ export class IgxCarouselComponent implements OnDestroy, AfterContentInit {
     @HostBinding('attr.id')
     @Input()
     public id = `igx-carousel-${NEXT_ID++}`;
-
-    /**
-     * Returns the `aria-label` of the carousel.
-     *
-     * ```typescript
-     * let carousel = this.carousel.ariaLabel;
-     * ```
-     *
-     */
-    @HostBinding('attr.aria-label')
-    public ariaLabel = 'carousel';
-
     /**
-     * Returns the `tabIndex` of the carousel component.
+     * Returns the `role` attribute of the carousel.
      * ```typescript
-     * let tabIndex =  this.carousel.tabIndex;
+     * let carouselRole =  this.carousel.role;
      * ```
      *
      * @memberof IgxCarouselComponent
      */
-    @HostBinding('attr.tabindex')
-    get tabIndex() {
-        return 0;
-    }
+    @HostBinding('attr.role') public role = 'region';
+
+    /** @hidden */
+    @HostBinding('attr.aria-roledescription')
+    public roleDescription = 'carousel';
+
+    /** @hidden */
+    @HostBinding('attr.aria-labelledby')
```

Comment:
Let's set this aria attr only when the label is visible - which is happening when the total slides are more than the `maximumIndicatorsCount` (5)